### PR TITLE
Update ViennaRNA Package to upstream version 2.6.2

### PR DESCRIPTION
**Pull request by RaumZeit**
This build will disable the inclusion of RNAxplorer, since it would introduce a lot of additional dependencies. RNAxplorer might be enabled again in one of the future release

Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by bernt-matthias** [comment]
> This build will disable the inclusion of RNAxplorer, since it would introduce a lot of additional dependencies. RNAxplorer might be enabled again in one of the future release

Maybe recipes with multiple outputs are something that could help, e.g. https://github.com/bioconda/bioconda-recipes/blob/edc88ae4047a2192691b959fb1374bbb66663611/recipes/openms-meta/meta.yaml#L44

**Comment by kbseah** [comment]
triggering new checks, osx problem should be resolved with #41526

**Comment by kbseah** [comment]
@RaumZeit the osx problem appears to be resolved now. do you want to request review and merge?

File: recipes/viennarna/build.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ fi
 if [ $PY3K -eq 1 ]; then
     VRNA_PYFLAGS="--without-python2"
 else
-    VRNA_PYFLAGS="--without-python"
+    VRNA_PYFLAGS="--with-python2 --without-python"
```
Comment (bernt-matthias) [inline recipes/viennarna/build.sh:13]: I **guess** bioconda is building for py3 only anyway, or?

**Review by bernt-matthias — APPROVED** [review]

File: recipes/viennarna/build.sh (hunk 1 above)
Comment (dpryan79) [reply recipes/viennarna/build.sh:13]: Yup, python 3.8-3.10 at the moment only

**Comment by RaumZeit** [comment]
Great! Thanks a lot for rebuild and merge!

File: recipes/viennarna/build.sh (hunk 1 above)
Comment (RaumZeit) [reply recipes/viennarna/build.sh:13]: Ok, so its safe to remove this part from the build script with the next release?!

Comment (bernt-matthias) [reply recipes/viennarna/build.sh:13]: Jep

**Pull request merged**